test_clouds/brazilus.json

```diff
@@ -0,0 +1,10 @@
+{
+	"Name": "ARMTest",
+	"EndpointActiveDirectoryResourceId": "https://management.core.windows.net/",
+	"EndpointGallery": "https://gallery.azure.com/",
+	"EndpointManagement": "https://brazilus.management.azure.com/",
+	"EndpointResourceManager": "https://brazilus.management.azure.com/",
+	"EndpointActiveDirectory": "https://login.windows.net/",
+	"EndpointActiveDirectoryGraphWindowsId": "https://graph.windows.net/",
+	"EndpointVmImageAliasDoc": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-compute/quickstart-templates/aliases.json"
+}
```

Comment:
Still keep this in public? @tjprescott?

Reply:
No no. This should only be in the private repo.